docs/Formula-Cookbook.md

```diff
@@ -66,11 +66,11 @@ This creates `$(brew --repository)/Library/Taps/homebrew/homebrew-core/Formula/f
 
 ```ruby
 class Foo < Formula
-  desc ""
-  homepage ""
+  desc "Fill in a one-line description of your formula"
```

Comment:
Again: I think it's either worth removing this template or disabling this rule because I think it was better before